modules/social_features/social_mailer/social_mailer.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Social Mailer'
+type: module
+description: 'Social Mailer implementation'
+core_version_requirement: ^9
```

Comment:
Looking to D10 compatibility, isn't be better to add " || ^10".

Reply:
Yeah sure, I can also add the 10, double checking if we are actually d10 compatible